### PR TITLE
README: list all packages and fix ordering

### PR DESCRIPTION
**Pull request by muesli**
Included `daemon` and `login1` packages in README and fixed the sort order to match the enumeration of packages at the top of the file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,10 +20,9 @@ An example HTTP server using socket activation can be quickly set up by followin
 
 https://github.com/coreos/go-systemd/tree/master/examples/activation/httpserver
 
-## Journal
+## systemd Service Notification
 
-Using the pure-Go `journal` package you can submit journal entries directly to systemd's journal, taking advantage of features like indexed key/value pairs for each log entry.
-The `sdjournal` package provides read access to the journal by wrapping around journald's native C API; consequently it requires cgo and the journal headers to be available.
+The `daemon` package is an implementation of the sd_notify protocol. It can be used to inform systemd of service start-up completion, watchdog events, and other status changes.
```
Comment (lucab) [inline README.md]: It would nice to link "sd_notify protocol" to https://www.freedesktop.org/software/systemd/man/sd_notify.html#Description

Comment (muesli) [reply README.md]: Done!

**Comment by lucab** [comment]
LGTM. Can you please squash the changes in a single commit?

**Comment by muesli** [comment]
Happy to do that for you, but I think you could also do this from GitHub's PR interface now. The "Merge" button on this page should be a dropdown now, where you can also pick "Squash and Merge" (unless that is disabled project-wide).

Anyway, happy to squash it for you!

**Comment by muesli** [comment]
@lucab Squashed and pushed.

**Comment by lucab** [comment]
Thanks! Yes, squashing/rebasing is disabled across our repositories, and in general we prefer not to alter commits (e.g. because they are likely signed).

**Pull request merged**